7.1_kfc_merge_files_add_aml_identifier/kfc.yaml

```diff
@@ -1,6 +1,7 @@
-amlIdentifier:
+moduleIdentifier:
   moduleName: Merge every n files together cdd3b0ba
-  version: 0.0.1
+  moduleVersion: 0.0.1
+  namespace: test/test
```

Comment:
I don't think `test/test` is a valid namespace. The first section of namespace should be a domain name, such as `example.com`.

Reply:
What if an company or organization doesn't have a domain name?

Reply:
Maybe he or she could use github.com/name/ as the prefix of the domain name

Reply:
I'm not sure github.com is a good choice. Why the organization name needs to be a domain name?

Reply:
It seems that golang package name and docker image name are taking this manner. We want to follow this because it "looks mordern"